hs_geo_raster_resource/migrations/custom_migration_for_raster_meta_update_20160512.py

```diff
@@ -0,0 +1,186 @@
+from __future__ import unicode_literals
+
+import os
+import shutil
+import logging
+import tempfile
+import subprocess
+import xml.etree.ElementTree as ET
+
+from django.db import migrations
+from django.core.files.uploadedfile import UploadedFile
+
+from django_irods.storage import IrodsStorage
+from hs_core import hydroshare
+from hs_core.hydroshare.utils import resource_modified
+from hs_geo_raster_resource.models import RasterResource
+from hs_geo_raster_resource import raster_meta_extract
+
+
+def migrate_tif_file(apps, schema_editor):
+    log = logging.getLogger()
+    istorage = IrodsStorage()
+
+    start_migrate = True
+    check_success = False
+
+    copy_res_fail = []
+    vrt_update_fail = []
+    vrt_update_success = []
+    meta_update_fail = []
+    meta_update_success = []
+
+    # check whether the migration has been executed or not
+    for res in RasterResource.objects.all():
+        try:
+            scimeta_path = "{}/data/resourcemetadata.xml".format(res.short_id)
```

Comment:
@gantian127 Line 36 above may not work after my hydroshare user zone work gets deployed since the resource may reside in another federated zone with the path containing the zone name info. Since this migration will run only one time initially before my federation zone work gets merged, I will give +1 with this note for future reference.